fontbe/src/stat.rs

```diff
@@ -64,11 +62,10 @@ impl Work<Context, AnyWorkId, Error> for StatWork {
 }
 
 fn make_stat(static_metadata: &StaticMetadata) -> Stat {
-    let reverse_names: HashMap<_, _> = static_metadata
-        .names
-        .iter()
-        .map(|(key, name)| (name.as_str(), key.name_id))
-        .collect();
+    // If possible, reuse a matching name with the smallest ID to follow fonttools:
+    // https://github.com/fonttools/fonttools/blob/d5aec1b9/Lib/fontTools/otlLib/builder.py#L2884-L2886
+    // https://github.com/fonttools/fonttools/blob/d5aec1b9/Lib/fontTools/ttLib/tables/_n_a_m_e.py#L326-L329
```

Comment:
nit: move the comment to the method rather than the call location

Reply:
I hummed and hawed before homing these originally as each table in fonttools often has its own individual `name` strat depending on when it was implemented. No strong opinion either way tho, so happy to move, and I've pushed a commit now :+1: